google-cloud-firestore/src/test/java/com/google/cloud/firestore/WriteBatchTest.java

```diff
@@ -329,4 +336,37 @@ public void deleteDocument() throws Exception {
     CommitRequest commitRequest = commitCapture.getValue();
     assertEquals(commit(writes.toArray(new Write[] {})), commitRequest);
   }
+
+  @Test
+  public void bulkCommit() throws Exception {
+    BatchWriteResponse.Builder response = BatchWriteResponse.newBuilder();
+    response.addWriteResultsBuilder().getUpdateTimeBuilder().setSeconds(0).setNanos(1);
+    response.addWriteResultsBuilder();
+    response.addStatusBuilder().setCode(0).build();
+    response.addStatusBuilder().setCode(14).build();
+    doReturn(ApiFutures.immediateFuture(response.build()))
+        .when(firestoreMock)
+        .sendRequest(
+            batchWriteCapture.capture(),
+            Matchers.<UnaryCallable<BatchWriteRequest, BatchWriteResponse>>any());
+
+    List<Write> writes = new ArrayList<>();
+    batch.set(documentReference, LocalFirestoreHelper.SINGLE_FIELD_MAP);
+    writes.add(set(LocalFirestoreHelper.SINGLE_FIELD_PROTO));
+
+    batch.create(documentReference, LocalFirestoreHelper.SINGLE_FIELD_MAP);
+    writes.add(create(LocalFirestoreHelper.SINGLE_FIELD_PROTO));
+
+    assertEquals(2, batch.getMutationsSize());
+
+    List<BatchWriteResult> batchWriteResults = batch.bulkCommit().get();
+
+    assertEquals(Timestamp.ofTimeSecondsAndNanos(0, 1), batchWriteResults.get(0).getWriteTime());
+    assertEquals(Status.OK, batchWriteResults.get(0).getStatus());
```

Comment:
Nit (you may ignore): I would feel more natural to compare the status before comparing the time.

Reply:
makes sense -- flipped.